community/kernel/src/test/java/org/neo4j/kernel/impl/api/TransactionRepresentationCommitProcessTest.java

```diff
@@ -120,69 +109,10 @@ public void shouldCloseTransactionRegardlessOfWhetherOrNotItAppliedCorrectly() t
         verify( transactionIdStore, times( 1 ) ).transactionClosed( eq( txId ), anyLong(), anyLong() );
     }
 
-    @Test
-    public void shouldThrowWhenIndexUpdatesValidationFails() throws IOException
-    {
-        // Given
-        IndexUpdatesValidator indexUpdatesValidator = mock( IndexUpdatesValidator.class );
-        RuntimeException error = new UnderlyingStorageException( new IndexCapacityExceededException( 10, 10 ) );
-        when( indexUpdatesValidator.validate( any( TransactionRepresentation.class ) ) ).thenThrow( error );
-
-        StorageEngine storageEngine = mock( StorageEngine.class );
-        TransactionRepresentationCommitProcess commitProcess = new TransactionRepresentationCommitProcess(
-                mock( TransactionAppender.class ),
-                storageEngine,
-                indexUpdatesValidator );
-
-        try
-        {
-            // When
-            commitProcess.commit( mock( TransactionToApply.class ), CommitEvent.NULL, INTERNAL );
-            fail( "Should have thrown " + TransactionFailureException.class.getSimpleName() );
-        }
-        catch ( TransactionFailureException e )
-        {
-            // Then
-            assertEquals( Status.Transaction.ValidationFailed, e.status() );
-        }
-    }
-
-    @Test
-    public void shouldSuccessfullyCommitTransactionWithNoCommands() throws Exception
```

Comment:
hm.. test name is `shouldSuccessfullyCommitTransactionWithNoCommands`, and i suppose that transaction does not contain any index updates and that case should still work right? or i missing something?